rewrite-java-17/src/main/java/org/openrewrite/java/isolated/ReloadableJava17ParserVisitor.java

```diff
@@ -1824,6 +1834,21 @@ private <J2 extends J> List<JRightPadded<J2>> convertAll(List<? extends Tree> tr
         return converted;
     }
 
+    private <J2 extends J> List<JRightPadded<J2>> convertAll(List<? extends Tree> trees,
```

Comment:
Should the method above this not call this new overloaded variant?

Reply:
Yes, we could do that to remove some duplication.